tail.sh

```diff
@@ -357,6 +357,80 @@ sleep $BOOT_DELAY
   echo "$APP" >$psmgr
 ) &
 
+sleep $BOOT_DELAY
+
+(
+  APP=jupiter-brain
+  SITE=com
+  INFRA=macstadium
+  PAPERTRAIL_GROUP="08 - MacStadium"
+  PAPERTRAIL_PROGRAM="jupiter-brain-$ENV-$SITE"
+  PAPERTRAIL_GROUP_SUFFIX=''
+  JUPITER_BRAIN_DATASET="jupiter-brain"
```

Comment:
@igorwwwwwwwwwwwwwwwwwwww suggested we keep this in the worker dataset and filter by app name - fewer datasets would make navigating honeycomb easier and we could also correlate data across the apps.
So could you please remove the $JUPITER_BRAIN_DATASET and use the previously declared $HONEYCOMB_DATASET? 😸

Reply:
Fixed, I think!